pinot-core/src/test/java/org/apache/pinot/queries/FastFilteredCountTest.java

```diff
@@ -123,11 +122,16 @@ public void setUp()
     driver.init(segmentGeneratorConfig, new GenericRowRecordReader(records));
     driver.build();
 
-    IndexLoadingConfig indexLoadingConfig = new IndexLoadingConfig();
-    indexLoadingConfig.setInvertedIndexColumns(new HashSet<>(Arrays.asList(CLASSIFICATION_COLUMN, SORTED_COLUMN)));
-    indexLoadingConfig.setTextIndexColumns(Collections.singleton(TEXT_COLUMN));
-    indexLoadingConfig.setJsonIndexColumns(Collections.singleton(JSON_COLUMN));
-    indexLoadingConfig.setRangeIndexColumns(Collections.singleton(INT_RANGE_COLUMN));
+    List<FieldConfig> fieldConfigs = new ArrayList<>();
+    fieldConfigs.add(
+        new FieldConfig(TEXT_COLUMN, FieldConfig.EncodingType.DICTIONARY, FieldConfig.IndexType.TEXT, null, null));
+
+    TableConfig tableConfig = new TableConfigBuilder(TableType.OFFLINE).setTableName(RAW_TABLE_NAME)
+        .setInvertedIndexColumns(Arrays.asList(CLASSIFICATION_COLUMN, SORTED_COLUMN))
```

Comment:
(minor) Change all `Arrays.asList()` to `List.of()` which has slightly better performance